Smart wallet mobile SDK 0.1.0 doc update (#82)
coinbase/smart-wallet-docs: docs/pages/guides/react-native-integration.mdx

You can see more detailed instructions on setting up deeplinks in the [Expo docs](https://docs.expo.dev/guides/deep-linking/).

<Callout type="warning" >

> When instantiating SDK, `appDeeplinkUrl` is required to establish your app's identity, use 'https://' link for production. If a custom scheme `myapp://` is provided, user will see a banner warning about the insecure connection.

Reviewer: Couple of minor grammar adjustments

```suggestion
When instantiating the SDK, `appDeeplinkUrl` is required to establish your app's identity. Use 'https://' links for production. If a custom scheme `myapp://` is provided, users will see a banner warning about the insecure connection.
```

The `appDeeplinkUrl` is required and should match the deeplink URL you set up earlier. 

<Callout type="success" >

> A new config `appCustomScheme` is introduced to remove the Done screen after signing.

Reviewer: ```suggestion
The `appCustomScheme` config option can be used to remove the Done screen after signing.
```

Simply import the `createConnectorFromWallet` function and pass in the wallet you want to use to wagmi config.

<Callout type="success" >
A new config `appCustomScheme` is introduced to remove the Done screen after signing.

> </Callout>

Reviewer: This looks the same as above. Redundant?

Author: we provide 2 interfaces here, some user may jump right into the second one, would like to emphasize in both section
